INSTALL.md

```diff
@@ -7,92 +7,123 @@ which includes this INSTALL.md file, is denoted $(CERTIFIER).
 The Ubuntu 20.04 install guide can be found in
 [Ubuntu Install](./Doc/install-certifier-Ubuntu-20.04.md).
 
-Read the Guide in the Doc directory to understand the basic nomenclature
-and programs you will build and use.
+Read the [Guide](Doc/Guide.pdf) in the Doc directory to understand the
+basic nomenclature and programs you will build and use.
 
 
-Certifier API
--------------
+# Certifier API
 
-The certifier API is in the src directory.  To compile and run the
-certifier API tests:
+The certifier API is in the src/ directory.
 
-  cd $(CERTIFIER)/src
-  make clean -f certifier_tests.mak
-  make -f certifier_tests.mak
-  ./certifier_tests.exe [--print_all=true]
+## Setup steps
 
+You will need to install the following packages:
```

Comment:
I found that I needed to do all these pre-installs in order to get my 1st simple_app working correctly. 

I think listing this up-front will help the next person's on-boarding a bit smoother.